ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -271,6 +282,22 @@ export class PrepareMigrationComponent implements OnInit {
     )
   }
 
+  openDataflowForm() {
+    let dialogRef = this.dialog.open(DataflowFormComponent, {
+      width: '30vw',
+      minWidth: '400px',
```

Comment:
are these the same dimensions as other forms ? Ideally this sizing should come from CSS

Reply:
Yes they are the same. I've followed the same convention as other forms for now, but agree that sizing should come from the CSS.
Will look into it as a follow up change.